bidwire/bidwire_settings.py

```diff
@@ -2,24 +2,25 @@
 
 from bid import Bid
 from document import Document
-from notifiers.cityofboston_notifier import CityOfBostonNotifier
+from notice import Notice
+from notifiers.cityofboston_notifier import CityOfBostonNotifier, CityOfBostonNoticeNotifier
 from notifiers.commbuys_notifier import CommBuysNotifier
 from notifiers.knox_tn_agendas_notifier import KnoxCoTNAgendaNotifier
 from notifiers.massgov_notifier import MassGovNotifier
 from notifiers.memphis_council_calendar_notifier import MemphisCouncilCalNotifier
-from scrapers.cityofboston_scraper import CityOfBostonScraper
+from scrapers.cityofboston_scraper import CityOfBostonScraper, NoticesScraper
 from scrapers.commbuys_scraper import CommBuysScraper
 from scrapers.knox_tn_agendas_scraper import KnoxCoTNAgendaScraper
 from scrapers.massgov_eopss_scraper import MassGovEOPSSScraper
 from scrapers.memphis_council_calendar_scraper import MemphisCouncilCalScraper
 
 POSTGRES_ENDPOINT = os.environ.get('POSTGRES_ENDPOINT', 'localhost')
 SENDGRID_API_KEY = os.environ.get('SENDGRID_API_KEY')
-DEBUG_EMAIL = os.environ.get('DEBUG_EMAIL', 'bidwire-logs@googlegroups.com')
-ADMIN_EMAIL = "bidwire-admin@googlegroups.com"
+DEBUG_EMAIL = os.environ.get('DEBUG_EMAIL', 'zacharyspector@gmail.com')
+ADMIN_EMAIL = "zacharyspector@gmail.com"
```

Comment:
We probably don't want to have your personal email as the admin email or the default debug email. 😉